src/amo/containers/CategoryList.js

```diff
@@ -0,0 +1,30 @@
+import { compose } from 'redux';
+import { asyncConnect } from 'redux-connect';
+import { connect } from 'react-redux';
+
+import Categories from 'amo/components/Categories';
+import { loadCategoriesIfNeeded } from 'core/utils';
+
+
+export function mapStateToProps(state, ownProps) {
+  const addonType = ownProps.params.addonType.replace(/s$/, '');
```

Comment:
That's actually a fair idea. I'm gonna file an issue because I think it requires an additional error message/test and I'd rather split it out, but 👍